app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -1,3 +1,5 @@
+@import "colors";
```

Comment:
You could import `colors.scss` in `_application.scss` before `bootstrap-complete.scss`. (and the same for `mobile.scss`)